packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,160 @@
+
+# Tomcat
+session.timeout=${env:SESSION_TIMEOUT:60}
+session.config.tracking.mode.cookie=${env:SESSION_CONFIG_TRACKING_MODE_COOKIE:true}
+
+
+## common-base confs mapping
+
+nuxeo.server.http.port=${env:NUXEO_SERVER_HTTP_PORT:8080}
+nuxeo.server.http.maxThreads=${env:NUXEO_SERVER_HTTP_MAXTHREADS:20}
+nuxeo.server.http.acceptCount=${env:NUXEO_SERVER_HTTP_ACCEPTCOUNT:1000}
+nuxeo.server.http.connectionUploadTimeout=${env:NUXEO_SERVER_HTTP_CONNECTIONUPLOADTIMEOUT:60000}
+nuxeo.server.ajp.enabled=${env:NUXEO_SERVER_AJP_ENABLED:false}
+nuxeo.server.ajp.port=${env:NUXEO_SERVER_AJP_PORT:8009}
+nuxeo.server.ajp.secretRequired=${env:NUXEO_SERVER_AJP_SECRETREQUIRED:true}
+nuxeo.server.ajp.secret=${env:NUXEO_SERVER_AJP_SECRET:changeme}
+nuxeo.server.https.port=${env:NUXEO_SERVER_HTTPS_PORT:0}
+nuxeo.server.tomcat_admin.host=${env:NUXEO_SERVER_TOMCAT_ADMIN_HOST:localhost}
+nuxeo.server.tomcat_admin.port=${env:NUXEO_SERVER_TOMCAT_ADMIN_PORT:8005}
+nuxeo.server.tomcat_error.show_report=${env:NUXEO_SERVER_TOMCAT_ERROR_SHOW_REPORT:false}
+nuxeo.server.tomcat_error.show_server_info=${env:NUXEO_SERVER_TOMCAT_ERROR_SHOW_SERVER_INFO:false}
+nuxeo.server.jvmRoute=${env:NUXEO_SERVER_JVMROUTE:nuxeo}
+NuxeoContainer.autoactivation=${env:NUXEOCONTAINER_AUTOACTIVATION:false}
+# HSTS configuration is applied when HTTPS is enabled, unless the following property is false
+nuxeo.server.hsts.enabled=${env:NUXEO_SERVER_HSTS_ENABLED:true}
+nuxeo.server.hsts.maxage=${env:NUXEO_SERVER_HSTS_MAXAGE:2592000}
+nuxeo.server.hsts.includesubdomains=${env:NUXEO_SERVER_HSTS_INCLUDESUBDOMAINS:false}
+nuxeo.server.hsts.preload=${env:NUXEO_SERVER_HSTS_PRELOAD:false}
+
+# Mail server
+mail.store.protocol=${env:MAIL_STORE_PROTOCOL:pop3}
+mail.store.host=${env:MAIL_STORE_HOST:localhost}
+mail.store.port=${env:MAIL_STORE_PORT:110}
+mail.store.user=${env:MAIL_STORE_USER:anonymous}
+mail.store.password=${env:MAIL_STORE_PASSWORD:secret}
+# Mail transport
+mail.transport.protocol=${env:MAIL_TRANSPORT_PROTOCOL:smtp}
+mail.transport.host=${env:MAIL_TRANSPORT_HOST:localhost}
+mail.transport.port=${env:MAIL_TRANSPORT_PORT:25}
+mail.transport.user=${env:MAIL_TRANSPORT_USER:anonymous}
+mail.transport.password=${env:MAIL_TRANSPORT_PASSWORD:secret}
+mail.transport.usetls=${env:MAIL_TRANSPORT_USETLS:false}
+mail.transport.auth=${env:MAIL_TRANSPORT_AUTH:false}
+# User who will receive mail
+mail.user=${env:MAIL_USER:nobody}
+# Enable debugging output from the javamail classes
+mail.debug=${env:MAIL_DEBUG:false}
+mail.from=${env:MAIL_FROM:noreply@nuxeo.com}
+
+# disable ES usage
+elasticsearch.enabled=false
+
+
+## aws confs mapping
+
+nuxeo.aws.accessKeyId=${env:NUXEO_AWS_ACCESSKEYID:}
+nuxeo.aws.secretKey=${env:NUXEO_AWS_SECRETKEY:}
+nuxeo.aws.region=${env:NUXEO_AWS_REGION:}
+
+
+## s3binaries confs mapping
+
+# mandatory
+nuxeo.s3storage.bucket=${env:NUXEO_S3STORAGE_BUCKET:}
+nuxeo.core.binarymanager=${env:NUXEO_CORE_BINARYMANAGER:org.nuxeo.ecm.core.storage.sql.S3BinaryManager}
```

Comment:
Do we need it?

Reply:
dunno, i guess we will see

Reply:
If we change the default in Nuxeo (which will come) I'd rather not have this deployment package override it. I think it's best to remove this.

Reply:
you mean the binarymanager one @efge ? ok this one probably does not need to be mentioned here :+1:

Reply:
Yes I was thinking of the `nuxeo.core.binarymanager` property.

Reply:
done